### PR TITLE
Adds TimeGate endpoint

### PR DESCRIPTION
**Pull request by machawk1**

### CODE REVIEW

File: ipwb/util.py [hunk 1]
```diff
@@ -146,10 +146,30 @@ def datetimeToRFC1123(digits14):
         newLocale = 'en_US.utf8'
 
     locale.setlocale(locale.LC_TIME, newLocale)
+
+
+def datetimeToRFC1123(digits14):
```
Comment (ibnesayeed) [inline ipwb/util.py]: Name of this method should reflect that it is the opposite of `rfc1123ToDigits14()`

Comment (machawk1) [reply ipwb/util.py]: I agree. Stand by.

Comment (machawk1) [reply ipwb/util.py]: Updated in latest. Investigating a TravisCI tests issue that may have the unit test names hard-coded. Please continue the review.

**Comment by ibnesayeed** [comment]
To deal with various datetime formats, I would define the following functions:

```python
def parseRFC1123(RFC1123_datetime_string):
  return native_datetime_object

def parse14Digit(14Digit_datetime_string):
  return native_datetime_object

def to14Digit(native_datetime_object):
  return "14Digit_datetime_string"

def toRFC1123(native_datetime_object):
  return "RFC1123_datetime_string"
```

These can be reused in many ways as individual helper functions and also as composite functions for example to convert HTTP DateTime to 14-digit format, either two functions can be called inline in sequence or wrapped in another function as following:

```python
def RFC1123To14Digit(RFC1123_datetime_string):
  return to14Digit(parseRFC1123(RFC1123_datetime_string))
```

**Comment by machawk1** [comment]
@ibnesayeed Ok, that could be a good future optimization for clarity and reuse. Let's get it right first then make it fast/pretty.

**Comment by ibnesayeed** [comment]
Add `Vary: Accept-Datetime` response header to TG endpoint. This is a critical requirement.

**Comment by machawk1** [comment]
🎉 Closes #105

**Pull request merged**